mteb/tasks/Classification/swe/SwedishSentimentClassification.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 1024
```

Comment:
Is this necessary? I would prefer if you just wrote 1024 twice

Reply:
I just was asked to do it this way on previous PRs.

Reply:
alright then